### ISSUE TITLE
Instant mode switch unchecked = active

### ISSUE THREAD

**Issue by openffchrome** [BotoX/xiaomi-m365-firmware-patcher#55]
hello i have the Instant mode switch unchecked but it's active, do you know why? it's a bug?

my settings :  https://m365.botox.bz?version=DRV143&normal_max_speed=28&eco_max_speed=5&motor_start_speed=4.98&motor_power_constant=46000&boot_with_eco=on

**Comment by MarlonMrN** [comment]
Do you know for sure how that feature should work?

**Comment by openffchrome** [comment]
of course, you can try the settings, instant switch patch is not checked but firmware act like ithe patch is applied

**Comment by Nojevah** [comment]
This option switches from normal mode to eco mode when driving (or vice versa). Are you sure that's what happening ? If this option is disabled, you can change mode only when stopped.

I don't even remembered if I activated this setting last time I created my firmware (3 months ago) so I can't confirm or deny. I know it was working some months ago.
